### PR TITLE
adding important algorithms

### PR DESCRIPTION
**Pull request by shubh3483**

### CODE REVIEW

File: Algorithms/GRAPHS/DFS &BFS/BFS&DFS.CPP [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#include<iostream.h>
```
Comment (Manvityagi) [inline Algorithms/GRAPHS/DFS &BFS/BFS&DFS.CPP:1]: Graphs already has a Traversals folder in which DFS and BFS are already present. 
Please update your PR and delete your DFS/BFS.

File: Algorithms/GRAPHS/Dijkstra/DIJKSTRA.CPP [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#include<iostream>
```
Comment (Manvityagi) [inline Algorithms/GRAPHS/Dijkstra/DIJKSTRA.CPP:1]: Dijkstra is also already present in the Single Source Shortest distance folder in graphs.

File: Algorithms/GRAPHS/Floyd Warshall/floydwarshall.cpp [hunk 3]
```diff
@@ -0,0 +1,69 @@
+#include<iostream>
```
Comment (Manvityagi) [inline Algorithms/GRAPHS/Floyd Warshall/floydwarshall.cpp:1]: Floyd Warshall is already present in the Minimum spanning tree folder in graphs.

**Review by Manvityagi — CHANGES_REQUESTED** [review]
@shubh3483 Please deleted the repeated algorithms from your PR. I will merge it thereafter.

**Comment by shubh3483** [comment]
Hey! I had deleted all the pre existing algorithms
Thanks

**Review by Manvityagi — APPROVED** [review]

**Pull request merged**